src/DeviceInput/Implementations/webDeviceInputSystem.ts

```diff
@@ -609,6 +621,8 @@ export class WebDeviceInputSystem implements IDeviceInputSystem {
                         this._unregisterDevice(DeviceType.Touch, deviceSlot);
                     }
                 }
+                // Clear all active touches
+                this._activeTouchIds = [];

```

Comment:
Avoid recreating the array
```suggestion
                this._activeTouchIds.length = 0;
```